app/src/main/java/org/mozilla/fenix/utils/ClearableEditText.kt

```diff
@@ -46,7 +46,9 @@ class ClearableEditText @JvmOverloads constructor(
      * Displays a clear icon if text has been entered.
      */
     override fun onTextChanged(text: CharSequence?, start: Int, lengthBefore: Int, lengthAfter: Int) {
-        val drawable = if (shouldShowClearButton(lengthAfter)) {
+        //lengthAfter has inconsistent behaviour when there are spaces in the entered text, so we'll use text.length
```

Comment:
Small style nit: can you add a space after the `//`?

Reply:
Sure thing!